tests/demos/property.test.js

```diff
@@ -4,7 +4,9 @@ import Property from "@demos/components/property";
 
 describe("demos/components/property", () => {
   it("renders itself correctly", () => {
-    const component = renderer.create(<Property />).toJSON();
-    expect(component).toMatchSnapshot();
+    const component_single = renderer.create(<Property name="Button"/>).toJSON();
+    const single = expect(component_single).toMatchSnapshot();
```

Comment:
there is no need to create a variable here. same two lines below.

also, we never use `snake_case`, only `camelCase` for variable names.